### DISCUSSION TITLE
Add new sections in non-tool pyproject.toml?

### DISCUSSION THREAD

**Discussion by geehdev** [python-poetry/poetry#9273] [Q&A] [answered]
# Hi, I'm using Poetry for dependency management only and I have a project that contains two TOML files.

Config.toml file that contains two tables the values ​​of the custom table are always changed
```
[default]
background_color = ""
buttons_color = ""

[custom]
background_color = "" 
button_color = "" 
```
Is there a problem if I transfer the default and custom sections from config.toml to the pyproject.toml file generated by Poetry?

```
[tool.poetry]
package-mode = false

[tool.poetry.dependencies]
python = "^3.10"

[default]
background_color = ""
buttons_color = ""

[custom]
background_color = "" 
button_color = "" 

[build-system]
requires = ["poetry-core"]
build-backend = "poetry.core.masonry.api"
```

**Comment by Secrus** [comment, marked as answer]
`pyproject.toml` spec is defined in [PEP 518](https://peps.python.org/pep-0518/#specification) and [PyPA spec](https://packaging.python.org/en/latest/specifications/pyproject-toml/#pyproject-toml-spec). By this spec, you should put your tables under `[tool.your_tool_name]`. Poetry has nothing to do with it.
